benchmark/cluster/vgg16/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:2.7.14
+RUN pip install -U kubernetes opencv-python &&   apt-get update -y &&   apt-get install -y iputils-ping libgtk2.0-dev
+# NOTE: By default CI built wheel packages turn WITH_DISTRIBUTE=OFF,
+#       so we must build one with distribute support to install in this image.
+RUN pip install paddlepaddle
```

Comment:
Does this `pip install` is redundant? Move the dataset download after line12 ?

Reply:
No, in order to make debugging faster, lines below changes much, and download dataset is slow, so add this line.